File: src/main/java/testes/SuiteGeral.java
package testes;

import org.junit.AfterClass;
import org.junit.runner.RunWith;
import org.junit.runners.Suite;
import org.junit.runners.Suite.SuiteClasses;

import _core.Capabilities;

@RunWith(Suite.class)
@SuiteClasses({


})

public class SuiteGeral {

	@AfterClass
	public static void finalizar() {

		Capabilities.finalizarAppiumDrivery();
		
	}

}

File: src/test/java/qa/reinaldo/SuiteGeral.kt
package qa.reinaldo

import org.junit.AfterClass
import org.junit.runner.RunWith
import org.junit.runners.Suite

@RunWith(Suite::class)
@Suite.SuiteClasses(*[Runner::class])
object SuiteGeral {
    @AfterClass
    fun finalizar() {
        _core.Capabilities.Companion.finalizarAppiumDrivery()
    }
}